arm-network/2016-12-01/swagger/virtualNetworkGateway.json

```diff
@@ -1600,6 +1611,130 @@
       ],	  
       "description": "Response for GetConnectionSharedKey API service call"
     },
+    "IpsecPolicy": {
+      "properties": {
+        "sALifeTimeSeconds": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The Security Association lifetime in seconds for a site to site VPN tunnel."
+        },
+        "sADataSizeKilobytes": {
```

Comment:
note that the properties in here have to be called *precisely* as provided/expected on the wire by your service! if your service sends/expects `SALifeTimeSeconds`, name it that way :-)